### PR TITLE
Add blank storageClassName to provisioner PVC & PV template

### PR DESCRIPTION
**Pull request by wongma7**
First 2 commits are followups to address comments from https://github.com/openshift/openshift-ansible/pull/3691

3rd commit is to fix https://bugzilla.redhat.com/show_bug.cgi?id=1464384

### CODE REVIEW

**Comment by michaelgugino** [comment]
Closing due to inactivity.

**Comment by wongma7** [comment]
Don’t close it, review it. Or find somebody else to review it. There is a bugzilla

File: roles/openshift_provisioners/tasks/generate_clusterrolebindings.yaml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ---
 - name: Generate ClusterRoleBindings
-  template: src=clusterrolebinding.j2 dest={{mktemp.stdout}}/templates/{{obj_name}}-clusterrolebinding.yaml
+  template:
+    src: clusterrolebinding.j2
+    dest: {{mktemp.stdout}}/templates/{{obj_name}}-clusterrolebinding.yaml
```
Comment (michaelgugino) [inline roles/openshift_provisioners/tasks/generate_clusterrolebindings.yaml]: Missing spaces and quotes here.  This line should be

```yaml
dest: "{{ mktemp.stdout }}/templates/{{ obj_name }}-clusterrolebinding.yaml"
```

Other files need similar changes.

**Comment by wongma7** [comment]
@michaelgugino fixed and squashed into commit 1.

Thank you

**Comment by michaelgugino** [comment]
This is currently at the top of the merge queue and flaked for no reason.  Merging.

**Pull request merged**